operator/internal/server/handlers.go

```diff
@@ -18,6 +22,17 @@ func (ws *WebServer) WriteResponse(w http.ResponseWriter, httpRes HTTPResponse)
 	}
 }
 
+func (ws *WebServer) ProxyToKubeAPI(res http.ResponseWriter, req *http.Request) {
+	targetUrl, _ := url.Parse("http://localhost:8231")
+	// create the reverse proxy
+	proxy := httputil.NewSingleHostReverseProxy(targetUrl)
+
+	req.URL.Path = "/" + strings.Join(strings.Split(req.URL.Path, "/")[2:], "/")
```

Comment:
can you test <host>/k8s and see if you get an index error?

Reply:
![image](https://user-images.githubusercontent.com/29277992/126316852-5568aa46-fd93-4b15-8535-a56babbd6200.png)
Seems to work fine